Jenkinsfile

```diff
@@ -153,7 +153,7 @@ pipeline {
                         source ${PWD}/.envrc
                         set -x
                         unset HCF_PACKAGE_COMPILATION_CACHE
-                        rm -f scf-*-amd64-*.zip
+                        rm -f scf-*amd64*.zip
```

Comment:
After ew-reading the zip PR, How about `...-*-amd64.zip` as pattern? A bit more specific.